src/domain/individualCustomer/customerInfoCard/CustomerInfoCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import Text from '../../../common/text/Text';
+import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
+import Paragraph from '../../../common/paragraph/Paragraph';
+
+interface Props {
+  firstName: string;
+  lastName: string;
+  primaryAddress?: {
+    address: string;
+    postalCode: string;
+    city: string;
+  } | null;
+  phone?: string | null;
+  email?: string | null;
+  comment?: string | null;
+}
+
+const CustomerInfoCard: React.SFC<Props> = ({
+  firstName,
+  lastName,
+  primaryAddress,
+  phone,
+  email,
+  comment,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card title={t('individualCustomer.customerInformation.title')}>
+      <Text as="h4" size="m">
+        {t('individualCustomer.customerInformation.applicantPersonalInfo')}
+      </Text>
+      <Paragraph>
+        <LabelValuePair
+          label={t('individualCustomer.customerInformation.firstNames')}
```

Comment:
Should this be first name or first names?

Reply:
I don't really know 🤔in Abstract it's `Etunimet`

Reply:
well, the field is called `firstName` in the backend, but in reality a lot of customers will have all their first names in there

Reply:
I'd prefer to follow strictly backend spec over UI design, but I'm willing to let this slide if you prefer to keep it like this.